conformance/base/manifests.yaml

```diff
@@ -91,9 +91,15 @@ spec:
   selector:
     app: infra-backend-v1
   ports:
-  - protocol: TCP
+  - name: first-port
+    protocol: TCP
     port: 8080
     targetPort: 3000
+  - name: third-port
+    protocol: TCP
+    appProtocol: kubernetes.io/ws
+    port: 8082
+    targetPort: 3000
```

Comment:
Do we really want to use the same targetPort for both TCP and WS traffic?

Reply:
What is your concern about using the same targetPort?

Reply:
I have no real concern, I was just wondering if this choice was backed by a specific reason. If not, I think we should prefer not to use the same back-side port for multiple front-side ports.

Reply:
ok, sounds good to me 👍